dispatcher_plugin_nb2workflow/products.py

```diff
@@ -417,4 +417,17 @@ def __init__(self,
                          extra_metadata=extra_metadata)
 
     def get_html_draw(self):
-        return self.dispatcher_data_prod.get_html_draw()  # type: ignore
\ No newline at end of file
+        # If there is only one extension of the fits file, it is a primary, and therefore it is the only one suitable to be an image.
+        # The other option is specific to INTEGRAL products, where we want to display the significance map among other extensions.
```

Comment:
is that really only integral? The schema is a bit more general maybe.

Reply:
I'm not sure, @ferrigno can comment on this